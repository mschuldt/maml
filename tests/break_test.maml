arduino = Arduino(desktop=True)

@block(once)
def test():
	i = 3
	while i > 0:
		x = 2
		while x > 0:
			x-=1
			print(x)
			if i == 2:
				break
			print(i)
		i-=1
	die(1)

arduino.send(test)
