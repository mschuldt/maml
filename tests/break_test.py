arduino = Arduino(desktop=True)

@block
def test():
	i = 3
	while i > 0:
		x = 2
		while x > 0:
			x-=1
			print_i(x)
			if i == 2:
				break
			print_i(i)
		i-=1
	die()

arduino.send(test)