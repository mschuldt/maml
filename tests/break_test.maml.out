Initializing avm...
Ready.

1
3
0
3
1
1
1
0
1
bye.
